scraibe/transcript_exporter.py

```diff
@@ -5,6 +5,110 @@
  
 ALPHABET = [*"abcdefghijklmnopqrstuvwxyz"]
 
+WHISPER_HALLUCINATIONS={
```

Comment:
Please move the transcriptions into an external file (like hallucinations.json ore something)

Reply:
Add the file to the 'package_data' in setup.py file

Reply:
added the list of known hallucinations to external python file, but forgot to add that into init.py.
Does setup.py need it to be mentioned in init?